src/Engine/ProtoCore/Parser/atg/Associative.atg

```diff
@@ -1904,9 +1904,8 @@ Associative_String<out ProtoCore.AST.AssociativeAST.AssociativeNode node>
                 node = new ProtoCore.AST.AssociativeAST.StringNode() 
                 { 
                     value = GetEscapedString(t.val.Length <= 2 ? "" : t.val.Substring(1, t.val.Length - 2)),
-                    line = t.line,
-                    col = t.col
                 }; 
+                NodeUtils.SetNodeLocation(node, t);
```

Comment:
Yea, it was the endline thats not set. It is being checked when generating lines in the CBN.
I removed the line and col because they were redundant